src/transformation/Transformation.ts

```diff
@@ -254,6 +255,15 @@ class Transformation {
     return this.addAction(action);
   }
 
+  /**
+   * @doc
+   * @description Adds a page or frame from a document
```

Comment:
here also would change to extracts just so naming is more clear to whoever is using

Reply:
Changed